libafl_bolts/src/serdeany.rs

```diff
@@ -254,7 +254,20 @@ macro_rules! create_serde_registry_for_trait {
                 where
                     T: $trait_name,
                 {
-                    self.map.insert(unpack_type_id(TypeId::of::<T>()), t);
+                    let id = unpack_type_id(TypeId::of::<T>());
+                    assert!(
+                        unsafe {
+                            REGISTRY
```

Comment:
This makes calling `insert_boxed` concurrently unsound and hence makes the function need `unsafe`.. :/

Reply:
Thinking about it some more, it's non-mutable, and there is no safe way to alter the Registry after, so should be fine, nevermind :)